src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -225,6 +238,10 @@ public static void setStandaloneConfigIgnoredToRepoConfigs(
         }
     }
 
+    public static boolean isAnyRepoFindingPreviousAuthors(List<RepoConfiguration> configs) {
+        return configs.parallelStream().anyMatch(RepoConfiguration::isFindingPreviousAuthorsPerformed);
```

Comment:
Let's use `stream()` instead of `parallelStream()` to standardize with the rest of the codebase.

Reply:
I have yet to make a PR for this branch, I'm not sure which branch to direct the PR to since the branch is non existent on the remote repository. Aside from that I need the commit hashes in my the branch above to be preserved (as it is used in the tests), is there any way you can ensure that?